docs/UserGuide.md

```diff
@@ -138,13 +138,25 @@ Format: `search [n/NAME] [i/ID] [g/SEX] [s/SPECIES] [b/BREED] [db/DOB] [da/DOA]`
 * Searches for animals whose attributes contain the specified search values in the prefix filter.
 * The search is case-insensitive. e.g `n/pookie` will match animals named `Pookie`
 * The order of prefixes does not matter. e.g. both `n/Pookie i/1234` and `i/1234 n/Pookie `will match animals named `Pookie` with ID `1234`
-* There must be at least 1 prefix specified.
+* Searching for `n/ear` will not match animals whose name contains `ear` like `Bear`, it will only search for animals whose names are exactly `ear`.
 * If searching by more than 1 prefix, the animal's attribute has to match all the specified fields e.g. `search b/dog n/tofu` will only return a dog named tofu and not any other dog or any pet named tofu
+* Any characters or words after `search` and before any recognized prefixes will be ignored.
+* Any unrecognized prefixes will be ignored and will not be treated as a keyword to search for.
```

Comment:
For these 2 points, do you think putting in examples of such cases would help?

Reply:
Yeah I agree, I'll add it in